pkg/google/gke/pricing_map_test.go

```diff
@@ -644,13 +640,17 @@ func TestGeneratePricingMap(t *testing.T) {
 		},
 	} {
 		t.Run(tc.name, func(t *testing.T) {
-			pm, err := GeneratePricingMap(tc.skus)
+			pricingMap := &PricingMap{
+				Compute: make(map[string]*FamilyPricing),
+				Storage: make(map[string]*StoragePricing),
+			}
```

Comment:
Specifically not calling `NewPricingMap(...)` because that method will also call `populate`. I'm specifically trying to avoid having to mock out the billing client.